packages/svelte/src/internal/client/proxy/proxy.js

```diff
@@ -42,6 +48,56 @@ export function proxy(value) {
 	return value;
 }
 
+/**
+ * @template {StateObject} T
+ * @param {T} value
+ * @param {Map<T, Record<string | symbol, any>>} already_unwrapped
+ * @returns {Record<string | symbol, any>}
+ */
+export function unwrap(value, already_unwrapped = new Map()) {
```

Comment:
I don't think we need to expose this
```suggestion
function unwrap(value, already_unwrapped = new Map()) {
```